app/models/member.rb

```diff
@@ -5,7 +5,8 @@ class Member < ActiveRecord::Base
 
   scope :managers, conditions: { manager: true }
   scope :nonmanagers, conditions: 'manager IS NULL or manager = 0'
-
+  scope :recent_community_owner, ->(current_user) {where(user_id: current_user.id, manager: true,
+                         created_at: DateTime.now.in_time_zone.to_date.beginning_of_day..DateTime.now.in_time_zone.to_date.end_of_day)}
```

Comment:
Line is too long. [135/80]<br>Align the elements of a hash literal if they span more than one line.<br>Expression at 9, 135 should be on its own line.